sorc/checkout_upp.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -eu
+
+# Get the root of the cloned WAFS directory
+readonly DIR_ROOT=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/.." && pwd -P)
+
+# Checkout upp code
+git -c submodule."sorc/upp.fd".update=checkout submodule update --init --recursive
```

Comment:
I think this does not need to be this way.  The upp repo is not private.
This should be sufficient.
```suggestion
cd "${DIR_ROOT}/sorc"
git submodule update --init --recursive 
```

Reply:
When I make a separate PR about UPP, I will include this change. Thank you.